src/main/java/com/rocketden/main/config/WebSocketConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.rocketden.main.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+
+	@Override
+	public void configureMessageBroker(MessageBrokerRegistry config) {
+		config.enableSimpleBroker("/topic");
+		config.setApplicationDestinationPrefixes("/app");
```

Comment:
What do these two lines do?

Reply:
Good question -- nothing important right now, I just removed them! They have to do with setting preferences and prefixes in paths for sending and receiving messages from the server, but since I'm not doing any of that yet in this PR, I just removed them. You can read more about them at [this Spring tutorial](https://spring.io/guides/gs/messaging-stomp-websocket/) -- do `Command + f` for "configureMessageBroker".